package.json

```diff
@@ -102,5 +108,10 @@
       "@semantic-release/npm",
       "@semantic-release/github"
     ]
+  },
+  "imports": {
+    "#cli/*.js": "./src/*.js",
+    "#tests/*.js": "./test/*.js",
+    "#tests/*.json": "./test/*.json"
```

Comment:
Couldn't we use the `@/*`: `./src/*` syntax that is common in vite projects. It feels like a better standard. I'd do something like:

```
    "@/*.js": "./src/*.js",
    "@tests/*.js": "./test/*.js",
    "@tests/*.json": "./test/*.json"
```

If it's possible

Reply:
It's not; the use of `#` is enforced by Node itself - https://nodejs.org/docs/v20.16.0/api/packages.html#subpath-imports

Reply:
Ok